strawberry/schema/schema.py

```diff
@@ -104,6 +105,9 @@ def __init__(
                     self.schema_converter.from_schema_directive(type_)
                 )
             else:
+                if hasattr(type_, "_type_definition"):
+                    if type_._type_definition.is_generic:
```

Comment:
can we do this if in one line? :)

Reply:
Black will reformat an `and` across 4 lines.

Reply:
I could live with four lines. To make the whole thing slightly prettier we could use a `continue` and remove the `else`. That would reduce the indentation level by one. Just an idea tho.

Reply:
@patrick91 this PR overall looks good to me. Don't want to ignore your comment tho. Do you prefer 4 lines or 2 ifs? :)

Reply:
I'm fine as it is! Let's merge the PR 😊 Sorry @coady for the delay!